temperature_regulator/client.py

```diff
@@ -0,0 +1,8 @@
+import socket
```

Comment:
taka jeszcze ostatnia uwaga. Czy myślałeś może o używaniu typów https://docs.python.org/3/library/typing.html Ten projekt może być nieco większy więc może warto o tym pomyśleć

Reply:
To zależy jak duży będzie projekt, teraz nie potrafię tego jeszcze przewidzieć. Jest opcja aby mieszać statyczne i dynamiczne typowanie dzięki bibliotece mypy. Zobaczę czy będzie ona konieczna.

Reply:
Nie wiem w jakim środowisku piszesz ale np PyCharm wspiera typy out of the box. Ale spoko typy nie są czymś niezbędnym

Reply:
Zakładam, że PyCharm wspiera typy poprzez lintera co jednak nie wystarcza. Używam VSCode.